bin/vps-lib.sh

```diff
@@ -126,6 +126,25 @@ check_for_bins() {
   return 0
 }
 
+test_hosts_entry() {
+  host_ip="$1"
+  host_name="$2"
+  if [[ $# -ne 2 ]] ; then
+    echo "usage: $(funname) <IP> <hostname>"
```

Comment:
I searched the code and Google for `funname` but didn't find anything. Is this a typo of the bash variable [`$FUNCNAME`](https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html#index-FUNCNAME)? It seems like it's not since it's an executable. I'm not sure, but figured I'd ask.